src/components/Form/Form.js

```diff
@@ -5,6 +5,7 @@ import SaveBtn from './Button/SaveBtn'
 import ActivityInput from './Inputs/ActivityInput'
 import TimeInput from './Inputs/TimeInput'
 import CategorySelect from './Select/CategorySelect'
+
```

Comment:
Maybe put your magic string 'Choose category' into a const?

Reply:
Good idea! It made some part of my code less redundant.